hw/ip/sram_ctrl/rtl/sram_ctrl.sv

```
@@ -26,16 +26,20 @@ module sram_ctrl
   parameter int NumPrinceRoundsHalf                        = 3,
   // Number of outstanding TLUL transfers
   parameter int Outstanding                                = 2,
-  // Random netlist constants
-  parameter  otp_ctrl_pkg::sram_key_t   RndCnstSramKey   = RndCnstSramKeyDefault,
-  parameter  otp_ctrl_pkg::sram_nonce_t RndCnstSramNonce = RndCnstSramNonceDefault,
-  parameter  lfsr_seed_t                RndCnstLfsrSeed  = RndCnstLfsrSeedDefault,
-  parameter  lfsr_perm_t                RndCnstLfsrPerm  = RndCnstLfsrPermDefault,
-  parameter bit                         EnableRacl       = 1'b0,
-  parameter bit                         RaclErrorRsp     = EnableRacl,
+  // RACL configuration of this IP including the policy selection for the configuration registers
+  // and the SRAM memory window
+  parameter bit                         EnableRacl         = 1'b0,
+  // Enable single-bit error correction and error logging
+  parameter bit                         EccCorrection      = 0,
+  parameter bit                         RaclErrorRsp       = EnableRacl,
```

I am aware that this wasn't changed in this PR and I don't ask to change this now, but: is there a use case where we want `RaclErrorRsp` to be different from `EnableRacl`? If not, then `RaclErrorRsp` should be defined as `localparam` and there should be a trailing comment `// derived parameter`.

---

Those are 2 independent parameters. In the default configuration and it is totally fine to configure them differently. RACL supports two different error responses:
* RaclErrorRsp = 1: Return a TLUL error on a RACL violation
* RaclErrorRsp = 0: Silently drop writes and return all-zero on a RACL violation
The defaults are choseen such that when RACL is disabled, also the TLUL error behavior is disabled. Better safe than sorry, just another layer of defense that no behavior is changed. If RACL is enabled, the default behavior is then to return a TLUL error. But, as said above, it's totally fine to tie RaclErrorRsp to 0 when RACL is enabled.

---

Thanks for your clarification, this makes sense!